.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -130,7 +130,7 @@ jobs:
         # Just run go mod tidy and hope for the best.
         go mod tidy
         cd scanner
-        go build -trimpath -o bin/updater ./cmd/updater
+        go build -trimpath -buildvcs=false -o bin/updater ./cmd/updater
         go clean -cache -modcache
```

Comment:
I am wondering if the comment above is still relevant, and if we should not build the updater using `make`. [In the dev workflow](https://github.com/stackrox/stackrox/blob/master/.github/workflows/scanner-dev-vuln-update.yaml#L34-L35) we use `make`, but I am not sure that is supported in all releases, so I am OK leaving it as is if you prefer to.

Reply:
Will leave as-is for now so the vuln updates can be fixed asap